ArduCopter/Log.cpp

```diff
@@ -617,7 +617,7 @@ const struct LogStructure Copter::log_structure[] = {
 void Copter::Log_Write_Vehicle_Startup_Messages()
 {
     // only 200(?) bytes are guaranteed by AP_Logger
-    logger.Write_MessageF("Frame: %s", get_frame_string());
+    logger.Write_MessageF("Frame: %s, Type: %s", motors->get_frame_string(), motors->get_type_string());
```

Comment:
Do you think "Layout" might be better than "type"?

Reply:
Wouldn't that conflict with a lot of user-facing stuff: https://ardupilot.org/copter/docs/frame-type-configuration.html?highlight=frame%20type